src/admin/src/components/page/advisoryDashboard/AdvisoryDashboard.js

```diff
@@ -67,99 +71,113 @@ export default function AdvisoryDashboard({
+        (as) => as.code === "PUB"
+      );
+      if (publishedStatus && publishedStatus[0]) {
+        await cmsAxios
+          .get(
+            `/public-advisories?_advisoryStatus=${publishedStatus[0].id}&_limit=-1`
+          )
+          .then((res) => {
+            const result = res.data;
+            let publishedAdvisories = [];
+            result.forEach((ad) => {
+              publishedAdvisories = [
+                ...publishedAdvisories,
+                ad.advisoryNumber,
+              ];
+            });
+            if (isMounted.current) {
+              setPublishedAdvisories([...publishedAdvisories]);
+            }
+          })
+          .catch(() => {
+            setHasErrors(true);
+          });
+      }
+    }
+  };
+
+  const fetchPublicAdvisory = async (selectedParkId) => {
+    setIsGridLoading(true);
+    setHasErrors(false);
     let parkIdQuery =
       selectedParkId > 0 ? `&protectedAreas.id=${selectedParkId}` : "";
-    const response = await Promise.all([
+    await Promise.all([
       getManagementAreas(cmsData, setCmsData),
       cmsAxios.get(
         `public-advisory-audits?_limit=1500&_sort=advisoryDate:DESC${parkIdQuery}`,
         {
           headers: { Authorization: `Bearer ${keycloak.token}` },
         }
       ),
-    ]);
-
-    const managementAreas = response[0];
-    const publicAdvisories = response[1].data;
-
-    const getCurrentPublishedAdvisories = (cmsData, setCmsData) => {
-      const advisoryStatuses = getAdvisoryStatuses(cmsData, setCmsData);
-      if (advisoryStatuses) {
-        const publishedStatus = advisoryStatuses.filter(
-          (as) => as.code === "PUB"
-        );
-        if (publishedStatus && publishedStatus[0]) {
-          cmsAxios
-            .get(
-              `/public-advisories?_advisoryStatus=${publishedStatus[0].id}&_limit=-1`
-            )
-            .then((res) => {
-              const result = res.data;
-              let publishedAdvisories = [];
-              result.forEach((ad) => {
-                publishedAdvisories = [
-                  ...publishedAdvisories,
-                  ad.advisoryNumber,
-                ];
-              });
-              if (isMounted.current) {
-                setPublishedAdvisories([...publishedAdvisories]);
-              }
-            });
-        }
-      }
-    };
+    ])
+    .then((response) => {
+      getCurrentPublishedAdvisories(cmsData, setCmsData);
 
-    getCurrentPublishedAdvisories(cmsData, setCmsData);
+      const managementAreas = response[0];
+      const publicAdvisories = response[1].data;
 
-    const regionParksCount = managementAreas.reduce((region, item) => {
-      region[item.region.id] =
-        (region[item.region.id] || 0) + item.protectedAreas.length;
-      return region;
-    }, {});
+      const regionParksCount = managementAreas.reduce((region, item) => {
+        region[item.region.id] =
+          (region[item.region.id] || 0) + item.protectedAreas.length;
```

Comment:
Can be one line.

Reply:
Single line goes beyond 80 characters in one line hence it was written like that.

Reply:
done!